examples/kubernetes/pod_iam_impersonation/README.md

```diff
@@ -0,0 +1,3 @@
+## IRSA POD IAM IMPERSONATION
+
+TBC
```

Comment:
Do we have plan to fill this?

Reply:
Yes, this will serve as example/documentation of the feature. It will be ready in the next couple of days.

Reply:
Fixed